From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Sep 2021 10:07:46 +0100
Subject: build: Remove redundant AC_CONFIG_AUX_DIRS

There's an invocation of AC_CONFIG_AUX_DIR(build-scripts) at the top of
the file, which is sufficient. AC_CONFIG_AUX_DIRS is an undocumented,
internal version of AC_CONFIG_AUX_DIR that takes a whitespace-separated
list, instead of a single path to add to the list.

Newer versions of autoconf treat the argument to AC_CONFIG_AUX_DIRS
as being literal (they do not expand the shell variable), causing
autoreconf to fail. The argument to AC_CONFIG_AUX_DIR is documented
to be relative to $srcdir anyway, so there is no need to specify $srcdir
a second time.

Part of commit 614326ff "autotools build system updates" upstream.

Bug-Debian: https://bugs.debian.org/993157
Forwarded: not-needed, part of a larger commit upstream
---
 configure.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.in b/configure.in
index 240a322..02b97a4 100644
--- a/configure.in
+++ b/configure.in
@@ -41,7 +41,6 @@ AC_SUBST(LT_REVISION)
 AC_SUBST(LT_AGE)
 
 dnl Detect the canonical build and host environments
-AC_CONFIG_AUX_DIRS($srcdir/build-scripts)
 dnl AC_CANONICAL_HOST
 
 dnl Check for tools
